README.md

```diff
@@ -47,9 +48,10 @@ docker run --rm \
       [-p rtsp_password] \
       [-s rtsp_resolution] \
       [-f rtsp_framerate] \
+      [-i input]
       [-t] \
       [-d] \
-      [-i input]
+      [-g] gstreamer_pipeline \
```

Comment:
formatting should be `[-g gstreamer_pipeline]`

Reply:
☺️